### DISCUSSION TITLE
Hello everyone

### DISCUSSION THREAD

**Discussion by MerlinSchaefer** [OpenMined/courses#15] [Introduce Yourself]
Hey everyone,

I'm Merlin from Berlin,Germany.
Happy to start this journey with you and learn everything about the exiting topics of Privacy&AI.
Feel free to reach out to me for anything, always happy to connect with likeminded individuals! :)

**Comment by yemikifouly** [comment]
Hi Merlin! 👋 Welcome to OpenMined! Great to have you here! 🙂 At OpenMined we love people who enjoy connecting. I would strongly encourage you to comment on posts from the other students, answer questions posted if you think you know the answer and overall, just engaging with everyone here. By using a discussion board with this setup our hope is also to allow those enrolled in the courses to mingle and connect so don't hesitate! 🙂
